Source/file.swift

```diff
@@ -0,0 +1,5 @@
+class xscShortVariableName {
+  func   Xt(a:  Int) {
```

Comment:
Colon Violation: Colons should be next to the identifier when specifying a type. (colon)